tajo-core/pom.xml

```diff
@@ -262,9 +262,9 @@
       <artifactId>commons-codec</artifactId>
     </dependency>
     <dependency>
-      <groupId>org.dspace.dependencies</groupId>
-      <artifactId>dspace-geoip</artifactId>
-      <version>1.2.3</version>
+      <groupId>com.maxmind.geoip</groupId>
+      <artifactId>geoip-api</artifactId>
+      <version>1.2.14</version>
```

Comment:
Could you change to latest version? and this dependency scope should be changed to “provided”.